test/generator/case_values_auto_extractor_test.rb

```diff
@@ -0,0 +1,116 @@
+require_relative '../test_helper'
```

Comment:
`case_values_auto_extractor_test.rb` has too many `_auto_extractor`s in the filename